client/src/game/Draw.js

```diff
@@ -0,0 +1,6 @@
+export default function draw(context) {
```

Comment:
The filename is in PascalCase. So I thought, this is a React component.
I would rename it to `draw.js` or find a more generic name if you like to have multiple functions exported like the other functions `resizeCanvas` and `clearCanvas`.

Reply:
Thanks for the quick review 😍 
Thought that as well, but wasn't sure. It will be replaced with "drawPlayer.js" and "drawPlattforms.js" anyway 😉

Reply:
Good idea 👍 . I would update this PR to reduce the changes in upcoming PRs.